.github/ISSUE_TEMPLATE/NEW-REGISTRATION.yml

```diff
@@ -0,0 +1,69 @@
+name: "New Registration"
+description: Express interest in registering a new encoding
```

Comment:
> If would also like to reserve a prefix for compatibility, please file a separate issue in this repository using the "New Registration" issue template.

This one's a little better, but I'd still prefer this in PR form rather than this asking us to do the PR for them; I'm not really a user of github issue templates (nor particularly a fan! I did my time in JIRA hell already so I can do without the extreme structure), do we have the tools to shunt people toward a PR with all of these things filled out there?

Reply:
Will rework the semantics so that it's more of a placeholder.  Maybe something like "open this issue so we can give you timely feedback before you start, but please open a draft PR soon and work there until you can close the issue you just opened" would work?

Reply:
Does [the insertion of this checkbox](https://github.com/multiformats/multibase/pull/109/commits/d20580d00abac517cc412c20b95b53ecddd3195e) make it explicit enough?

Reply:
Yeah, I guess -- what I'd like to avoid here is a collection of poorly maintained opened issues with no clear path to getting them closed. If we permission people opening unactionable issues, then we can't take action and have no clear path to closing them.

I think it's fine to have issues opened flagging intent to register, but we have to acknowledge that there's a limit to how long those will be held.